cassandra/protocol_features.py

```diff
@@ -63,3 +69,9 @@ def parse_sharding_info(options):
                                             shard_aware_port, shard_aware_port_ssl)
 
 
+    @staticmethod
+    def parse_tablets_info(options):
+        tablets = options.get(TABLETS_ROUTING_V1) or None
+        if tablets is not None:
+            return True
+        return False
```

Comment:
nit: `options` is a dict? If so, then this function body can be replaced by `return TABLETS_ROUTING_V1 in options`.